### PR TITLE
Fix english (hopefully) and update some browsers!

### PR DESCRIPTION
**Pull request by Element21**
Also added millify to the 'points won' message. Please leave a comment if I made a stupid mistake! :bug:

### CODE REVIEW

**Comment by Tkd-Alex** [comment]
Thanks for your contributing :smile: 
I need a good English course :laughing: 

Where and how did you find the updated user-agent?

File: TwitchChannelPointsMiner/TwitchChannelPointsMiner.py [hunk 1]
```diff
@@ -190,12 +190,12 @@ def run(self, streamers: list = [], followers=False):
...
                 if self.ws_pool.ws.elapsed_last_ping() > 5:
                     logger.info(
-                        "The last ping was sent more than 5 minutes ago. Reconnect the WebSocket"
+                        "The last ping was sent more than 5 minutes ago. Reconnecting to the WebSocket..."
                     )
                     WebSocketsPool.handle_websocket_reconnection(self.ws_pool.ws)
 
     def end(self, signum, frame):
-        logger.info("CTRL+C Detected! Please wait, just a moment")
+        logger.info("CTRL+C Detected! Please wait just a moment!s")
```
Comment (Tkd-Alex) [inline TwitchChannelPointsMiner/TwitchChannelPointsMiner.py:198]: It's correct **!s** ?

File: TwitchChannelPointsMiner/classes/TwitchBrowser.py [hunk 2]
```diff
@@ -211,7 +211,7 @@ def __init_firefox(self):
             )
         else:
             logger.warning(
-                f"The path {self.settings.driver_path} is not valid. Use default path",
+                f"The path {self.settings.driver_path} is not valid. Using default path",
```
Comment (Tkd-Alex) [inline TwitchChannelPointsMiner/classes/TwitchBrowser.py:214]: This is related to Firefox. About Chrome?

**Review by Element21 — COMMENTED** [review]
> Thanks for your contributing 
> I need a good English course 
> 
> Where and how did you find the updated user-agent?

https://www.whatismybrowser.com/detect/what-is-my-user-agent on chrome and firefox on manjaro linux

**Comment by Element21** [comment]
Should I rebase to the current master branch?

**Comment by Tkd-Alex** [comment]
> > Thanks for your contributing
> > I need a good English course
> > Where and how did you find the updated user-agent?
> 
> https://www.whatismybrowser.com/detect/what-is-my-user-agent on chrome and firefox on manjaro linux

Ok ...
There is something wrong on my PC :laughing: 

image

**Pull request merged**